generic_config_updater/gu_common.py

```diff
@@ -113,9 +113,14 @@ def validate_sonic_yang_config(self, sonic_yang_as_json):
     def validate_config_db_config(self, config_db_as_json):
         sy = self.create_sonic_yang_with_loaded_models()
 
+        dup_lanes_platforms = [
```

Comment:
>dup_lanes_platforms [](http://example.com/codeflow?start=8&length=19)

you may move this block to inside `validate_lanes` function. Add a comment
```
# TODO: Move this attribute (platform with duplicated lanes in ports) to YANG models
```

Reply:
Could you add the TODO comment in my above comment?

Reply:
Added.